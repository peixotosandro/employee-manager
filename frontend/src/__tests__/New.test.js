import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import New from "../pages/New";

configure({ adapter: new Adapter() });

describe("Testing New Component", () => {
  it("should render properly.", () => {
    const wrapper = shallow(<New />);

    expect(wrapper).toMatchSnapshot();
  });
});
