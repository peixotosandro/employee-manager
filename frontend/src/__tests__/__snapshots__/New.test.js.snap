// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing New Component should render properly. 1`] = `ShallowWrapper {}`;
